content/blog/2020-sponsor-announcement.md

```diff
@@ -0,0 +1,39 @@
+Title: PyCon India 2020 is looking for Sponsors.
+Date: 2020-08-25 02:34 
+Author: Nikhil Maan(@Sc0rpi0n101)
+Category: 2020 
+Slug: announce-sponsorsership
+Summary: Want to give back to the Python community and have more reach among the community members? Here's your chance to put your organization in the limelight of the Python community by sponsoring the largest Python conference in India. 
+
+While the PyCon team is going full speed ahead to make PyCon India 2020 online an amazing conference and a great experience for everyone, here is **your** chance to help make the conference even more amazing.
+
+
```

Comment:
Looks good, but where's that graphic we talked about.

Reply:
Aren't we adding that to https://github.com/pythonindia/inpycon2020/pull/70

Reply:
Looks better with a photo and improves credibility. I think it be a good idea.